src/plugins/sound_canberra.py

```diff
@@ -14,7 +18,8 @@
 }
 
 
-def connect_plugin(category, label=None):
+def connect_plugin(category, label=None):    # pylint: disable=unused-argument
+    """Method helps to connect with plugin"""
```

Comment:
This is not a method. This function **implements** the entry point.

Reply:
I am not familiar with this code but I agree it's not a method.